client/verta/verta/runtime.py

```diff
@@ -212,7 +215,7 @@ def __enter__(self):
         """
         if hasattr(_THREAD, 'logs'):  # If logs attribute exists already.
             raise RuntimeError(
-                " Nesting an instance of verta.runtime.context() inside"
+                "nesting an instance of verta.runtime.context() inside"
```

Comment:
It burns my inner sense of grammar, but I have to break this habit and conform to the standard :)

Reply:
I'm with ya 😭